tests/sqlparser_bigquery.rs

```diff
@@ -84,9 +86,16 @@ fn parse_raw_literal() {
 
 #[test]
 fn parse_table_identifiers() {
-    fn test_table_ident(ident: &str, expected: Vec<Ident>) {
+    fn test_table_ident(ident: &str, canonical: Option<&str>, expected: Vec<Ident>) {
```

Comment:
can you please document what `canonical` represents in this context?

Reply:
Added!